docs/api/cloudlink/intro.md

```diff
@@ -8,4 +8,9 @@
     - The general request/response shape
 -->
 
```

Comment:
the points in the comment should be addressed either here or in a new pr, the current changes here aren't that great imo

Reply:
I was planning to do that when I get time, do you want me to wait to do the pull request until it's done?

Reply:
I will do that in a new pr when I get time.